docs/understand/hipgraph.rst

```diff
@@ -0,0 +1,91 @@
+.. meta::
+    :description: This chapter provides an overview over the usage of HIP graph.
+    :keywords: ROCm, HIP, graph, stream
+
+.. understand_HIP_graph:
+
+********************************************************************************
+HIP graph
+********************************************************************************
+
+HIP graphs are an alternative way of executing work on a GPU. It can provide
+performance benefits over repeatedly launching the same kernels in the standard
+way via streams.
+
+.. note::
+    The HIP graph API is currently in Beta. Some features can change and might
+    have outstanding issues. Not all features supported by CUDA graphs are yet
+    supported. For a list of all currently supported functions see the
+    :doc:`HIP graph API documentation<../doxygen/html/group___graph>`.
+
+Graph format
+================================================================================
+
+A HIP graph is made up of nodes and edges. The nodes of a HIP graph represent
+the operations performed, while the edges mark dependencies between those
+operations.
+
+The nodes can consist of:
+
+- empty nodes
+- nested graphs
+- kernel launches
+- host-side function calls
+- HIP memory functions (copy, memset, ...)
+- HIP events
+- signalling or waiting on external semaphores
+
+The following figure visualizes the concept of graphs, compared to using streams.
+
+.. figure:: ../data/understand/hipgraph/hip_graph.svg
+    :alt: Diagram depicting the difference between using streams to execute
+          kernels with dependencies, resolved by explicitly calling
+          hipDeviceSynchronize, or using graphs, where the edges denote the
+          dependencies.
+
+HIP graph advantages
+================================================================================
+
+The standard way of launching work on GPUs via streams incurs a small overhead
+for each iteration of the operation involved. For kernels that perform large
+operations during an iteration this overhead usually is negligible. However
```

Comment:
operations during an iteration this overhead is usually negligible. However

Reply:
Fixed